javascript/rest_API_return_top_1000_hits.js

```diff
@@ -4,34 +4,39 @@ This script makes a GET request to the /searches endpoint on the Analytics REST
 To get the top 1000 searches over the last 7 days.
 There is no API client for Analytics, so this script uses the JavaScript Requests library to make the call.
 */
-
+try {
 // Install the API client: https://www.algolia.com/doc/api-client/getting-started/install/javascript/?client=javascript
 const algoliasearch = require("algoliasearch");
```

Comment:
This import isn't needed. There's no api client for the analytics endpoint, we'll make direct calls to the REST api.

Reply:
Done @christopherjstock  - thanks!